From 3539aefb17239a40bdd05059a927d7b288573322 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Thu, 3 Dec 2015 09:51:56 +0000
Subject: [PATCH] fsl/uio: change UIO_FSL_SRAM kernel option from bool to
 tristate

Because UIO_FSL_SRAM depends on UIO subsystem, when UIO is built as
kernel module, L2CACHE should be changed to module state too.
So change L2CACHE kernel option from bool to tristate.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/uio/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/uio/Kconfig b/drivers/uio/Kconfig
index df737c9..004c15b 100644
--- a/drivers/uio/Kconfig
+++ b/drivers/uio/Kconfig
@@ -143,7 +143,7 @@ config UIO_PRUSS
 	  will be called uio_pruss.
 
 config UIO_FSL_SRAM
-	bool "Freescale L2CACHE for SRAM support"
+	tristate "Freescale L2CACHE for SRAM support"
 	depends on FSL_85XX_CACHE_SRAM=n
 	default n
 	help
-- 
2.0.2

